lib/TPP/BuilderUtils.cpp

```diff
@@ -67,8 +69,17 @@ Value getConstIndex(OpBuilder &builder, int value) {
 }
 
 Value createDenseTensor(OpBuilder &builder, TensorInitType initType,
-                        TensorType type, int seed) {
+                        TensorType type, int seed,
+                        std::optional<ModuleOp> module) {
   auto unkLoc = builder.getUnknownLoc();
+
+  if (module) {
```

Comment:
I'm not sure I understand what the module means here... Is it just a differentiation between tensor and memref on the TypeSwitch?

Reply:
Fair point, it was poor abstraction. Cleaned it up now.